src/components/Analysis/ContigViewer/index.tsx

```diff
@@ -271,9 +241,11 @@ const Contig: React.FC<ContigProps> = ({ contig }) => {
     updateTracks().then(() => setUpdatingTracks(false));
   }, [trackColorBys, igvBrowser]);
 
-  if (loading || loadingExtraAnnotations) return <Loading size="small" />;
+  // eslint-disable-next-line react/jsx-no-useless-fragment
+  if (loading || loadingExtraAnnotations) return <></>;
```

Comment:
I *think* you can just explicitly `return null` to have the same effect and avoid the eslint exception.

Reply:
@SandyRogers this has been done